src/Kriti/Parser/Token.hs

```diff
@@ -0,0 +1,148 @@
+{-# LANGUAGE RankNTypes #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Kriti.Parser.Token where
```

Comment:
The ADTs in this module should have some amount of documentation (`Symbol`, `Token`, `Accessor`, `ValueExt`).